src/app.rs

```diff
@@ -95,14 +88,24 @@ impl App {
     #[allow(clippy::missing_panics_doc)] // TODO
     #[instrument(name = "App::new", level = "debug")]
     pub async fn new(options: Options) -> EyreResult<Self> {
+        if let Ok(app) = Self::bootstrap(options.clone()).await {
+            Ok(app)
+        } else {
+            error!("Error when rebuilding tree from cache. Retrying with db cache busted.");
```

Comment:
This is a catch-all for a very specific error. We shouldn't wipe the database if there was an issue connecting to the Alchemy node.